runtime/compiler/env/j9method.cpp

```diff
@@ -764,11 +767,14 @@ TR_J9MethodBase::isBigDecimalConvertersMethod(J9ROMMethod * romMethod, J9ROMClas
 bool
 TR_J9MethodBase::isBigDecimalConvertersMethod(J9Method * j9Method)
    {
-   if (auto stream = TR::compInfoPT->getStream())
+#if defined(JITSERVER_SUPPORT)
+   if (TR::comp()->isOutOfProcessCompilation())
```

Comment:
In a future implementation we can avoid a message completely by using our caches on j9methods and j9classes

Reply:
Is this just a comment/note or is there anything that needs to be changed at the moment?

Reply:
I opened #7094 to address this in the future